src/firebase/firebase.js

```diff
@@ -2,15 +2,26 @@ import firebase from 'firebase/app';
 import 'firebase/firestore';
 import 'firebase/auth';
 
+// Due to env not working will put here for now cuz its blocking.
+const config = {
+    REACT_APP_FIREBASE_API_KEY: 'AIzaSyAXaRZRwMxLoe-b1BxqBnJjbdd7NZF-FEY',
+    REACT_APP_FIREBASE_AUTH_DOMAIN: 'cs6510-spr2021.firebaseapp.com',
+    REACT_APP_PROJECT_ID: 'cs6510-spr2021',
+    REACT_APP_STORAGE_BUCKET: 'cs6510-spr2021.appspot.com',
+    REACT_APP_MSG_SENDER_ID: '796249853280',
+    REACT_APP_APP_ID: '1:796249853280:web:56dc4cd01ba8bd8f7be205',
+    REACT_APP_MEASUREMENT_ID: 'G-NTJ3K8ST70',
```

Comment:
this is fine since firebase API keys can be public but please don't just do this. You are exposing secrets to the public. Can you please find another way to fix or ask someone who knows?

Reply:
Since this is blocking the crud testing. Can I resolve this in the next sprint?  For now, can we leave it undefined?

Reply:
I'd rather get it fixed asap since it's exposed. Don't worry about the sprint 😄

Reply:
Fixed.